custom-markers/index.ts

```diff
@@ -0,0 +1,197 @@
+import './style.css';
+import 'maplibre-gl/dist/maplibre-gl.css';
+import maplibregl from 'maplibre-gl';
+import {Deck} from '@deck.gl/core';
+import {BASEMAP, vectorQuerySource, VectorTileLayer} from '@deck.gl/carto';
+import {CollisionFilterExtension} from '@deck.gl/extensions';
+import Financial from './markers/Financial.svg';
+import Tourism from './markers/Tourism.svg';
+import Sustenance from './markers/Sustenance.svg';
+import Commercial from './markers/Commercial.svg';
+import Education from './markers/Education.svg';
+import Entertainment from './markers/Entertainment.svg';
+import Transportation from './markers/Transportation.svg';
+import Healthcare from './markers/Healthcare.svg';
+import Civic from './markers/Civic.svg';
+import Star from './markers/Others.svg';
+import Otehers from './markers/Others.svg';
+
+const apiBaseUrl = import.meta.env.VITE_API_BASE_URL;
+
+const accessToken = import.meta.env.VITE_API_ACCESS_TOKEN;
+
+const connectionName = 'carto_dw'
+
+const cartoConfig = {apiBaseUrl, accessToken, connectionName};
+
+const iconWidth = 15;
+
+const ICON_MAPPING = {
+  Financial: {
+    icon: Financial,
+  },
+  Tourism: {
+    icon: Tourism,
+  },
+  Sustenance: {
+    icon: Sustenance,
+  },
+  Commercial: {
+    icon: Commercial,
+  },
+  Education: {
+    icon: Education,
+  },
+  "Entertainment, Arts & Culture": {
+    icon: Entertainment,
+  },
+  Transportation: {
+    icon: Transportation,
+  },
+  Healthcare: {
+    icon: Healthcare,
+  },
+  'Civic amenities': {
+    icon: Civic,
+  },
+  Star: {
+    icon: Star,
+  },
+  Others: {
+    icon: Otehers,
+  }
+};
+
+const INITIAL_VIEW_STATE = {
+  latitude: 41.8097343,
+  longitude: -110.5556199,
+  zoom: 3,
```

Comment:
If we zoom into a city the viz should load faster and for this example is better. I would also limit the minZoom to avoid zoom out

Reply:
Done